acai-ios/SectionControllers/DetailViewSectionControllers/HeaderTextSectionController.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  HeaderTextSectionController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/7/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+class HeaderTextSectionController: ListSectionController {
+    var menuItem: MenuItem!
+    
+    override init() {
+        super.init()
+    }
+    
+    override func numberOfItems() -> Int {
+        return 1
+    }
+    
+    override func sizeForItem(at index: Int) -> CGSize {
+        guard let context = collectionContext, let _ = menuItem else {
+            return .zero
+        }
+        let width = context.containerSize.width
+        let height = estimatedHeightForText(text: menuItem.getSelectedSubOptionsText()) + 26 + 56 + 20
```

Comment:
Define constants for this class for the `26`, the `56`, and the `20` to make it clear what they represent.